### PR TITLE
LPS-145778 Remove jai-codec.jar and jai_core.jar

### PR DESCRIPTION
**Pull request by rafaprax**

### CODE REVIEW

**Comment by rafaprax** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:test

**Comment by rafaprax** [comment]
Hey @vicnate5, Can you please help me to scan these test results? Thx

File: portal-web/docroot/WEB-INF/shielded-container-web.xml [hunk 1]
```diff
@@ -249,10 +249,6 @@
 		<servlet-name>Combo Servlet</servlet-name>
 		<url-pattern>/combo/*</url-pattern>
 	</servlet-mapping>
-	<servlet-mapping>
```
Comment (tinatian) [inline portal-web/docroot/WEB-INF/shielded-container-web.xml:252]: @rafaprax You removed the servlet mapping for "Dynamic Resource Servlet" here which means corresponding servlet can not be accessed, you should also remove the servlet related code.

**Comment by tinatian** [comment]
Hi @rafaprax,
This commit https://github.com/liferay-core-infra/liferay-portal/pull/274/commits/a79a931dea17661b8c10b39b609a2399174c41dc contains lots of changes which is not easy to review and traced back when there is a problem, could you please separate it into two commits? One to remove the usages of sprite feature, the other one is to remove the sprite framework code.



Thanks.
Tina.

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
@rafaprax This looks good, I reviewed the older full test results and the new relevant results. No regressions found.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/112944

**Pull request closed without merging**